internal/serverinstall/k8s.go

```diff
@@ -1358,6 +1364,43 @@ func newServiceAccount(c k8sConfig) (*apiv1.ServiceAccount, error) {
 	}, nil
 }
 
+// newServiceAccountClusterRoleWithBinding creates the cluster role and binding necessary to create and verify
+// a nodeport type services.
+func newServiceAccountClusterRoleWithBinding(c k8sConfig) (*rbacv1.ClusterRole, *rbacv1.ClusterRoleBinding, error) {
+	roleName := "waypoint-runner"
```

Comment:
> roleName := "waypoint-runner"

Should this default to the same value as the `k8s-runner-service-account` flag (which defaults to `waypoint-runner`)?

Reply:
I've seen us do this both ways - sometimes we include the type of the resource in it's name, and sometimes we don't. My personal preference is not to include it - i.e. name a role `waypoint-runner` instead of `waypoint-runner-role`, because the resource type is almost always implied by the context.

The service account and role to have the same name won't cause a conflict.